src/library/components/resource-lightbox.js

```diff
@@ -448,25 +451,24 @@ var ResourceLightbox = Component({
-            {links.assign_collection ? <a className='portal-pages-secondary-button' href={`javascript: ${links.assign_collection.onclick}`} onClick={this.handleAddToCollectionClick}>{links.assign_collection.text}</a> : null}
-            {links.teacher_guide ? <a className='portal-pages-secondary-button' href={links.teacher_guide.url} target='_blank' onClick={this.handleTeacherGuideClick}>{links.teacher_guide.text}</a> : null}
+            {links.preview ? <a className='portal-pages-primary-button' href={links.preview.url} target='_blank' onClick={this.handlePreviewClick}>{previewButtonText}</a> : null}
+            {Portal.currentUser.isTeacher && resource.has_teacher_edition && !isCollection ? <a className='teacherEditionLink portal-pages-secondary-button' href={MakeTeacherEditionLink(resource.external_url)} target='_blank' onClick={this.handleTeacherEditionClick}>Teacher Edition</a> : null}
+            {links.teacher_resources && showTeacherResourcesButton && !isCollection ? <a className='teacherResourcesLink portal-pages-secondary-button' href={links.teacher_resources.url} target='_blank' onClick={this.handleTeacherResourcesClick}>{links.teacher_resources.text}</a> : null}
+            {links.assign_material && !isCollection ? <a className='portal-pages-secondary-button' href={`javascript: ${links.assign_material.onclick}`} onClick={this.handleAssignClick}>{links.assign_material.text}</a> : null}
+            {links.assign_collection && !isCollection ? <a className='portal-pages-secondary-button' href={`javascript: ${links.assign_collection.onclick}`} onClick={this.handleAddToCollectionClick}>{links.assign_collection.text}</a> : null}
+            {links.teacher_guide && !isCollection ? <a className='portal-pages-secondary-button' href={links.teacher_guide.url} target='_blank' onClick={this.handleTeacherGuideClick}>{links.teacher_guide.text}</a> : null}
           </div>
           <p className='portal-pages-resource-lightbox-description' dangerouslySetInnerHTML={this.longDescription()} />
           {resource.has_pretest ? <p className='portal-pages-resource-lightbox-description'>Pre- and Post-tests available</p> : null}
           {resource.saves_student_data === false ? <div className='portal-pages-resource-lightbox-no-save-warning'><strong>PLEASE NOTE:</strong> This resource can be assigned, but student responses will not be saved.</div> : null}
           {this.renderIncludedActivities()}
           <hr />
-          <h2>Requirements</h2>
-          {this.renderRequirements()}
-          {this.renderStandards()}
-          {this.renderLicense()}
-          {this.renderLearnMore()}
+          {!isCollection && this.renderRequirements()}
+          {!isCollection && this.renderStandards()}
+          {!isCollection && this.renderLicense()}
+          {!isCollection && this.renderLearnMore()}
```

Comment:
Perhaps an better way to do this is like this:
```
{!isCollection && <>
  {this.renderRequirements()}
  {this.renderStandards()}
  ...
</> }
```
Or like above make a `renderAssignableExtras` method.

@kswenson do you have an opinion this? 

I also looked into how hard it would be to turn all of these `renderBlah` methods into just `Blah` methods. So then they could be used with just `<Blah/>`. However these methods are using `this` to access the state. So I'm guessing this could break with any react magic that decides not to re-render Blah because nothing seems to have changed when really the state could have changed.

Reply:
I'm sure they could be converted to components and then any state that they're using could be passed in as props. But given that they're already written as functions that seems like busy-work at this point. Your suggestions of using a fragment or a `renderAssignableExtras()` seem like reasonable compromises.

Reply:
@kswenson @scytacki Thanks for the thoughts on this. I've made some changes in light of the above, but I'm wondering what you think about the below.

I couldn't get the fragment option to work for some reason, so I tried the other suggestion in both places. That worked, but I have a reservation about it. As I understand it, going that route means we'll have to wrap all the rendered items in a new containing element, introducing some superfluous markup. That's not the end of the world, but from my point of view it's unnecessary crud in the markup and I'd like to avoid it. But do you think the improvements to the React code are worth overlooking that?

Reply:
1. I don't think an additional `<div>` is likely to have any negative effect, so if the code is more readable/maintainable that seems like a reasonable tradeoff.
1. That said, in this case I don't think we have to make the tradeoff at all -- you should be able to return a fragment from the `renderAdditionalInfo()` function which should eliminate the extra div.

Reply:
Ah the problem is that the version of react doesn't support fragments. They were added in 16.2: https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html
But portal-pages is back on 15.6.2

Sorry for leading you down a rabbit hole. I don't think it is worth updating React just for this PR. So if the extra elements are annoying then I guess just put it back the way it was. 🤷‍♂

Reply:
It's OK with me if we leave the latest changes. I'll just try to remember to change to the fragment option when it becomes available. It's true the extra markup won't technically cause problems.

Reply:
Ah, I should have thought to check that before re-suggesting it. Good catch, Scott.